### PR TITLE
refactor: replace lodash templates #1810

### PR DESCRIPTION
**Pull request by urohit011**
Replaced lodash templates with native template literals, as mentioned in #1810.

### CODE REVIEW

**Comment by urohit011** [comment]
@jhermsmeier, made the changes you suggested.

File: lib/shared/messages.js [hunk 1]
```diff
@@ -32,14 +32,18 @@ module.exports = {
...
+      ].join(' ')
+    },
 
-    providePassword: _.template([
-      '<%= displayName %> needs special permissions to interact with raw devices. Please provide your password.'
-    ].join(' '))
+    providePassword: (displayName) => {
+      return [
+        `${displayName} needs special permissions to interact with raw devices. Please provide your password.`
+      ].join(' ')
```
Comment (jhermsmeier) [inline lib/shared/messages.js]: No need for the `.join()` here with only one string ;)

Comment (urohit011) [reply lib/shared/messages.js]: Oh yes, right.

Comment (urohit011) [reply lib/shared/messages.js]: Should I also remove enclosing array brackets?

Comment (urohit011) [reply lib/shared/messages.js]: Made the changes.

**Comment by jhermsmeier** [comment]
Can you please rebase this (we've made some conflicting changes recently)?

**Comment by urohit011** [comment]
Is it ok if I do it tomorrow?

**Comment by jhermsmeier** [comment]
Yeah sure; no rush – you can do it whenever you like!

**Comment by urohit011** [comment]
Done rebasing.

**Comment by jhermsmeier** [comment]
This is still missing a `Change-Type: patch` footer in the commit message

**Comment by Shou** [comment]
e.g. from https://github.com/resin-io/etcher/blob/master/lib/cli/etcher.js#L123

```diff
-    console.log(messages.info.flashComplete({
-      drive: results.drive,
-      imageBasename: path.basename(results.imagePath)
-    }))
+    console.log(messages.info.flashComplete(path.basename(results.imagePath), results.drive))
```

**Comment by Shou** [comment]
Hey @urohit011, I've rebased and updated this PR but since I can't write the changes here I will open a new PR and preserve you as the commit author. Apologies from us for taking so long to review your PR.

**Comment by urohit011** [comment]
Thanks for your response @Shou and for preserving me as the commit author :-)

**Comment by jhermsmeier** [comment]
Closed by https://github.com/resin-io/etcher/pull/2006

**Pull request closed without merging**